scripts/test-publish-docker-images

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Is this supposed to be in the update version commit?

Reply:
No this should not be.